api/spec/src/billing/invoice/bill.tsp

```diff
@@ -0,0 +1,407 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace Invoice.Bill;
+
+@summary("Invoice summary represents an extract of the invoice object")
+@friendlyName("BillingInvoiceSummary")
+model InvoiceSummary {
+  ...OpenMeter.InvoiceSummary; // OpenMeter schema extensions
+  type?: InvoiceType;
+
+  @summary("Used as a prefix to group codes.")
+  @visibility("read", "query")
+  series?: CBC.Code;
+
+  @summary("Sequential code used to identify this invoice in tax declarations.")
+  // NOTE: This is required in the original schema, but we can't enforce it here as we will only know this when the
+  // invoice is issued.
+  @visibility("read", "query")
+  code?: CBC.Code;
+
+  @summary("Currency for all invoice totals.")
+  currency?: Currency.Code;
```

Comment:
Reminder: We need to adopt in checkout session payload

Reply:
Yes, that should go into OpenMeter.InvoiceSummary.